### DISCUSSION TITLE
using breeze I am trying to build prod image and getting  gpg: keyserver receive failed: Network is unreachable . Server are behind proxy

### DISCUSSION THREAD

**Discussion by mohankumar-h** [apache/airflow#40423] [Q&A]
How to pass proxy for the breeze to reach.  

0.290 Try to receive GPG public key 0xF1656F24C74CD1D8 from hkps://keyserver.ubuntu.com
0.298 gpg: keybox '/tmp/tmp.yUjGFZbbwq/pubring.kbx' created
30.97 gpg: keyserver receive failed: Network is unreachable
30.97 Unable to receive GPG public key 0xF1656F24C74CD1D8 from hkps://keyserver.ubuntu.com
30.97 Try to receive GPG public key 0xF1656F24C74CD1D8 from hkps://pgp.surf.nl
95.42 gpg: keyserver receive failed: Connection timed out
95.42 Unable to receive GPG public key 0xF1656F24C74CD1D8 from hkps://pgp.surf.nl
95.43 gpg: WARNING: nothing exported

I need to pass the key to get it. How do I set proxy for gpg
ex: gpg --keyserver-options http-proxy=http://proxy.example.com:8080 --keyserver hkp://keyserver.ubuntu.com --recv-keys [key-id]

**Comment by potiuk** [comment]
There are likely few other things that will fail behind proxy. You should run teh builds in a network that does not need proxy. If you want to contribute a feature to add it (i.e. modify all the scripts to allow proxy - contributions to that are welcome).

**Comment by mohankumar-h** [comment]
So, no options are available to pass as  --build-arg https_proxy=proxy.domain:port ? same we pass during docker build?

**Reply by potiuk** [reply]
Generally speaking Breeze prod image build is used for Airlfow development, not for building image. Breeze prod build image is generally  fancy way to run `docker build` command. As mentioned above - if you would like to contribute some options for that - you are welcome. But for anything fancy you can always use `--dry-run` or `--verbose` options, copy-paste the command you will see and modify it as you fit.
